doc/source/data/dataset.rst

```diff
@@ -33,11 +33,24 @@ is already supported.
 Data Loading and Preprocessing for ML Training
 ----------------------------------------------
 
-Ray Datasets are designed to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
+Use Ray Datasets to load and preprocess data for distributed :ref:`ML training pipelines <train-docs>`.
 Compared to other loading solutions, Datasets are more flexible (e.g., can express higher-quality `per-epoch global shuffles <examples/big_data_ingestion.html>`__) and provides `higher overall performance <https://www.anyscale.com/blog/why-third-generation-ml-platforms-are-more-performant>`__.
 
-Ray Datasets are not intended as a replacement for more general data processing systems.
-:ref:`Learn more about how Ray Datasets work with other ETL systems <datasets-ml-preprocessing>`.
+Use Datasets for the last-mile bridge from ETL pipeline outputs to distributed applications and libraries in Ray. Do not use them as a replacement for more general data processing systems.
+Its utility is as the last-mile bridge from ETL pipeline outputs to distributed applications and libraries in Ray:
+
+.. image:: images/dataset-loading-1.png
+   :width: 650px
+   :align: center
+
+..
+  https://docs.google.com/presentation/d/1l03C1-4jsujvEFZUM4JVNy8Ju8jnY5Lc_3q7MBWi2PQ/edit
+
+You can also use Datasets seamlessly with Ray-integrated DataFrame libraries.
+
+.. image:: images/dataset-loading-2.png
+   :width: 650px
+   :align: center
```

Comment:
I am thinking of to drop this part. If we look at the images, they are mostly identical, except the scope of "Ray cluster". And to run Spark/Mars/etc on Ray is community-maintained, and not battle tested. So it should be safe to drop here.

Reply:
Removed